pom.xml

```diff
@@ -66,6 +66,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <automatic-module-name>--module-name-need-to-be-overriden--</automatic-module-name>
```

Comment:
All other property names use dots, so let's rename this to `automatic.module.name`.

Reply:
done